src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1101,6 +1102,11 @@ def update_connected_cluster(cmd, client, resource_group_name, cluster_name, htt
         cmd_helm_upgrade.extend(["--set", "global.isCustomCert={}".format(True)])
     if container_log_path is not None:
         cmd_helm_upgrade.extend(["--set", "systemDefaultValues.fluent-bit.containerLogPath={}".format(container_log_path)])
+    if enable_oidc_issuer:
+        if not auto_upgrade:
+            logger.warning("Please enable auto_upgrade to upgrade to latest agent version in order to enable this oidc issuer feature")
+        else:
+            cmd_helm_upgrade.extend(["--set", "systemDefaultValues.signingkeycontroller.oidcenabled={}".format(True)])
```

Comment:
we also need to handle the scenario of disabling the oidc issuer feature on day N. so, we should set 'systemDefaultValues.signingkeycontroller.oidcenabled' to false as well when the flag enable_oidc_issuer is passed as False.
If passed as None, then it should be kept as is